src/store/category/actions.js

```diff
@@ -23,3 +24,104 @@ export const fetchRecs = ({commit}, payload) => {
       _procError(err)
     })
 }
+export const fetchCategorys = ({commit}) => {
```

Comment:
Categorys  -> Categories

Reply:
can't...if change like that it'll have error in EtGrid
return dispatch(`${this.type}/fetch${upperFirst(this.type)}s`, payload)
if change to Categories must modify EtGrid again

Reply:
ETGrid already modify to have a aliasType name, so you could put  the aliasType="categorie"